polyfills/Array/from/polyfill.js

```diff
@@ -1,36 +1,104 @@
-Object.defineProperty(Array, 'from', {
-	configurable: true,
-	value: function from(source) {
-		// handle non-objects
-		if (source === undefined || source === null) {
-			throw new TypeError(source + ' is not an object');
+(function arrayFromPolyfill() {
```

Comment:
Leaking a global here.  Use anonymous fn, or for slightly simpler architecture, bring the parseIterable fn inside the Object.defineProperty getter fn